compose.yml

```diff
@@ -38,7 +38,7 @@ services:
       - "3000:3000"
     environment:
       PGRST_DB_URI: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB_NAME}
-      PGRST_DB_SCHEMAS: application, meta
+      # PGRST_DB_SCHEMAS: application, meta (shouldnt be necessary if I did this right, check init.sql)
```

Comment:
This change seems to break the postgrest API 
![image](https://github.com/WillTrem/UInnovate/assets/64431699/a12bde1d-2f62-4cda-bd5a-116aa1663aae)

Reply:
This should be fixed now with my last commit. You should be able to run 
 ./refresh_database.sh -f '/useCase1'
or 
 ./refresh_database.sh -f '/useCase2'

and have both schemas available in postman and the web app

Reply:
Works great! Thanks @witnick ! :)